database/migrations/2020_09_13_230104_replace_egg_json_with_yaml.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use Symfony\Component\Yaml\Yaml;
+use Illuminate\Database\Migrations\Migration;
+
+class ReplaceEggJsonWithYaml extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $eggs = DB::select('SELECT id, config_files, config_startup, config_logs FROM eggs;');
```

Comment:
Please import the DB facade here so we're not using the globally namespaced ones. 👍